src/audio/mixer.cpp

```diff
@@ -438,9 +437,11 @@ void Channel::SetGroup(int group)
 }
 
 Mixer::Mixer()
+	: deviceid(0)
+	, format({ 0 })
+	, effectbuffer(0)
+	, volume(1)
```

Comment:
The other initializer lists I can find are written on one line, but that's not readable for more than two members. I suggest to just place the commas at the end of the lines here, and the colon behind the method. Agreed @IntelOrca?